it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/BusinessIdentificationControllerISpec.scala

```diff
@@ -219,52 +209,34 @@ trait BusinessIdentificationControllerISpec extends BaseISpec with SessionDataMi
       metricShouldExistAndBeUpdated("Count-Subscription-AlreadySubscribed-HasEnrolment-AgentServicesAccount")
     }
 
-    BusinessIdentificationForms.validBusinessTypes.foreach { validBusinessTypeIdentifier =>
+    validBusinessTypes.foreach { validBusinessTypeIdentifier =>
       s"show check Agency Status page for valid businessTypeIdentifier: $validBusinessTypeIdentifier" in {
         val result = await(
-          controller.showBusinessDetailsForm(Some(validBusinessTypeIdentifier))(
+          controller.showBusinessDetailsForm(validBusinessTypeIdentifier)(
             authenticatedAs(subscribingCleanAgentWithoutEnrolments)))
         status(result) shouldBe 200
-        val bodyString = bodyOf(result)
-        bodyString should include(htmlEscapedMessage("businessDetails.title"))
-        bodyString should include(htmlEscapedMessage(s"businessDetails.label.utr.$validBusinessTypeIdentifier"))
-      }
-    }
 
-    s"redirect to obtain identifier again, when Invalid businessTypeIdentifier" in {
-      val result = await(
-        controller.showBusinessDetailsForm(Some("unCateredBusinessTypeIdentifier"))(
-          authenticatedAs(subscribingCleanAgentWithoutEnrolments)))
-
-      redirectLocation(result) shouldBe Some(routes.BusinessIdentificationController.showBusinessTypeForm().url)
-    }
-
-    s"redirect to obtain identifier again, empty businessTypeIdentifier" in {
```

Comment:
Similar deal. Is this the same as the removed test above ("no businessType parameter supplied")?
It's still good to capture the behaviour in a test for this case

Reply:
same deal as above